docs/release-notes/NuGet-6.0.md

```diff
@@ -0,0 +1,264 @@
+---
+title: NuGet 6.0 Release Notes
+description: Release notes for NuGet 6.0 including new features, bug fixes, and DCRs.
+author: zkat
+ms.author: kmarchan
+ms.date: 10/7/2021
+ms.topic: conceptual
+---
+
+# NuGet 6.0 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**<NuGetVersion>**](https://nuget.org/downloads) | [Visual Studio <VSYear> version <VSVersion>](https://visualstudio.microsoft.com/downloads/) | [<SDKVersion>](https://dotnet.microsoft.com/download/dotnet-core/<SDKMajorMinorVersionOnly>)<sup>1</sup> |
```

Comment:
NuGet - 6.0
Visual Studio 2019 version 16.11.4
Visual Studio 2022 version 17.0
.NET 5.0.402
.NET 6.0.0-rc.2


I believe these are right? May need to dig a little further though.